lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,23 @@
 
 def palindrome_permutation?(string)
+  hash = {}
+  leng = string.length
```

Comment:
Same feedback as given on `intersection` with the `hash` variable. With the `leng` variable it could be improved by specifying what it is the length of e.g. `input_length` or `string_length`.